ra/ra.go

```diff
@@ -2644,7 +2644,7 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 			return nil, err
 		}
 		newAuthzs = append(newAuthzs, pb)
-		ra.authzAges.WithLabelValues("NewOrder", pb.Status).Observe(0)
+		ra.authzAges.WithLabelValues("NewOrder", "pending").Observe(0)
```

Comment:
I think we can just leave this alone. If you'd rather hard-code I think that we can use core.StatusPending, maybe it's also a Stringer?